### ISSUE TITLE
All requests/data gone after restart

### ISSUE THREAD

**Issue by parmsib** [getinsomnia/insomnia#3413]
When I booted up my computer today and launched Insomnia, all my data was suddenly gone.
I had probably over a hundred requests in a bunch of nested folders, but it's all suddenly just missing :(

I tried exporting all Insomnia data to json, but it just resulted in this:

```
{"_type":"export","__export_format":4,"__export_date":"2021-05-21T09:38:08.287Z","__export_source":"insomnia.desktop.app:v2021.3.0","resources":[]}
```

OS: Windows 10 Pro 19042.985 Windows Feature Experience Pack 120.2212.2020.0
Insomnia version is in the export output above.

I don't have sync enabled (that I know of).

**Comment by dimitropoulos** [comment]
My heart skips a beat when I see something like this - I know I speak for the team when I say that we take data loss very seriously.  It's just about the worst-case scenario (one of the reasons we implemented sync!).  I'll be contacting you directly to see if we can debug it some time today (or soon) so that if there's any data still around maybe we can find it.

**Comment by hapi-cloud** [comment]
This has happened to me as well, when I had two instances of Insomnia open at the same time. 

Basic steps to recreate:
- open instance 1
- make some changes to instance 1 (add/update collections/documents)
- open instance 2
- make some changes to instance 2 (add/update collections/documents)
- close instance 2
- make some changes to instance 1 (add/update collections/documents)
- close instance 1
- open new instance
- changes made to instance 2 did not save

**Comment by filfreire** [comment]
Hi folks, I'm closing this one in favour of https://github.com/Kong/insomnia/issues/5167

- Avoid using more than one instance of Insomnia
- To retrieve your data, please refer to the workaround [posted here](https://github.com/Kong/insomnia/issues/5167#issuecomment-1317415546)
- A fix for this issue will be included on `2022.7.0-beta.5` and onwards.
